* fix detection of rage128/radeon on ppc (which seems to keep them disabled)

diff -r 143117b03502 vidix/drivers/radeon_vid.c
--- a/vidix/radeon_vid.c	Sun Apr 30 16:54:38 2006 +0200
+++ b/vidix/radeon_vid.c	Sun Apr 30 16:54:38 2006 +0200
@@ -1293,11 +1005,13 @@ int vixProbe( int verbose,int force )
 	dname = pci_device_name(VENDOR_ATI,lst[i].device);
 	dname = dname ? dname : "Unknown chip";
 	printf(RADEON_MSG" Found chip: %s\n",dname);
+#if 0
         if ((lst[i].command & PCI_COMMAND_IO) == 0)
         {
           printf("[radeon] Device is disabled, ignoring\n");
           continue;
         }
+#endif
 	memset(&besr,0,sizeof(bes_registers_t));
 	if(force > PROBE_NORMAL)
 	{
